src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1220,11 +1220,12 @@ def upload(cls, cmd, identifier, filepath, session_pool_endpoint, no_wait=False)
         return r.json()
 
     @classmethod
-    def show_file_content(cls, cmd, identifier, filename, session_pool_endpoint):
-        url_fmt = "{}/files/content/{}?identifier={}&api-version={}"
+    def show_file_content(cls, cmd, identifier, filename, path, session_pool_endpoint):
+        url_fmt = "{}/files/{}/content?{}identifier={}&api-version={}"
```

Comment:
Why change from `{}/files/content/{failename}?identifier={}` to `{}/files/{}/content?{}identifier=`??

Reply:
Oh, the path changed to `{}/files/{filename}/content?{}identifier=`